package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1957,6 +1957,13 @@ def _gen_initial_values(na, nr, ns):
         return np.zeros(na, dtype=bool)
 
 
+class RSChirality(AtomAttr):
+    "R/S chirality"
```

Comment:
Weird - are neither of these covered by tests?

Reply:
```suggestion
    """R/S chirality"""
```
Mostly to keep to the style.

Reply:
L1961 is a docstring, so I think this is just a codecov snafu